### PR TITLE
[15.O][FIX] hr_employee_phone_extension: Change fields in model hr.employee.base

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
TT47743

@Tecnativa

@pedrobaeza @victoralmau

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
:+1:

**Review by victoralmau — APPROVED** [review]

File: hr_employee_phone_extension/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -7,3 +7,4 @@
 * Anand Kansagra <kansagraanand@hotmail.com>
 * Saran Lim. <saranl@ecosoft.co.th>
 * aphon61bank <aphon61bank@gmail.com>
+* Carolina Fernandez <carolina.fernandez@tecnativa.com>
```
Comment (pedrobaeza) [inline hr_employee_phone_extension/readme/CONTRIBUTORS.rst]: Don't put your mail. You can use the two lines form

Comment (carolinafernandez-tecnativa) [reply hr_employee_phone_extension/readme/CONTRIBUTORS.rst]: Done!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**